src/Application/Factories/IAlarmMonitoredItemServiceInitializerFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿// SPDX-License-Identifier: MIT. 
+// Copyright Contributors to the Open Manufacturing Platform.
+
+using OMP.Connector.Domain;
+using OMP.Connector.Domain.Models.Telemetry;
+using OMP.Connector.Domain.Schema;
+using Opc.Ua.Client;
+
+namespace OMP.Connector.Application.Factories
+{
+    public interface IAlarmMonitoredItemServiceInitializerFactory
+    {
+        public MonitoredItem Initialize(
+            AlarmSubscriptionMonitoredItem monitoredItem,
+            IComplexTypeSystem complexTypeSystem,
+            TelemetryMessageMetadata telemetryMessageMetadata,
+            Session session);
```

Comment:
Can this not be our wrapper IOpcUaSession object to ease unit testing and reduce tight coupling?
(The opc ua foundation has still not merged the PR in that introduces the ISession abstraction ;( )